todo_app/app.py

```diff
@@ -91,59 +112,71 @@ def index():
             misWriter=True #for E2E testing
         else:
             mcurrent_user=current_user.username  
-            misWriter=(appsession.get('roles')==ROLES['writer'])     
+            misWriter=(appsession.get('roles')==ROLES['writer'])  
+        app.logger.info("Is Writer %s",misWriter)  
+        app.logger.debug("Index end")   
         return render_template('index.html',view_model=item_view_model,isWriter=misWriter,currentuser=mcurrent_user)
                    
 
     @app.route('/', methods=['POST'])
     @login_required
     @requires_roles('writer')
-    def add_item():   
+    def add_item():           
         title = request.form.get('title') 
+        app.logger.info("Add Item %s",title)   
         if title!='':
             session.Boards().add_item(title)
+            app.logger.info("Item Successfully Added")
```

Comment:
You could include the user id in the log for a nice audit trail, since there's no change log in the DB layer.